cppapi/server/tango_const.h.in

```diff
@@ -1051,6 +1050,12 @@ const char * const CmdArgTypeName[] = {
 	"Unknown"
 };
 
+/// Convert data types to strings
+TANGO_CONSTEXPR const char * DataTypeToString(int type)
```

Comment:
The vast majority of functions in Tango has snake_case names. Consider renaming to something like data_type_to_string(...) for consistency reasons.

Reply:
@mliszcz Thanks for the review. Good point with the naming. I've renamed it. And also fixed two more issues (not being inline for old compilers and more strict range checking.